variables.tf

```diff
@@ -1,17 +1,17 @@
 variable "groups" {
   description = "Enforce MFA for the members in these groups"
-  type = "list"
-  default = []
+  type        = list(string)
+  default     = []
 }
 
 variable "users" {
   description = "Enforce MFA for these users"
-  type = "list"
-  default = []
+  type        = list(string)
+  default     = []
 }
 
 variable "allow_password_change_without_mfa" {
   description = "Allow changing the user password without MFA"
-  type = "string"
-  default = "false"
+  type        = string
```

Comment:
Could we use the occasion to make this a `bool` here?

Reply:
sure, let me do it and test it locally. I'll send another commit